packages/big-design/src/components/Pagination/styled.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+
+import { StyleableButton } from '../Button/private';
+
+export const StyledButton = styled(StyleableButton)`
+  color: ${({ theme }) => theme.colors.secondary70};
```

Comment:
You'll probably also want to add state styles seen on the [`ButtonSubtle` component](https://github.com/bigcommerce/big-design/blob/master/packages/big-design/src/components/Button/styled.tsx#L186)